docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
+[chapter, role="xpack"]
+= Host isolation
+
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts. After hosts are isolated, they can still send data to {es} and {kib}.
+
+NOTE: To isolate and release hosts, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+
+[role="screenshot"]
+image::images/isolated-host.png[Shows a host that's been isolated]
+
+You can isolate a host via an alert attached to a case or from the Endpoints list. Once a host is successfully isolated, an *Isolated* status displays next to the *Agent Status* field, which you can view on the alert details flyout or Endpoints list table.
+
+TIP: If the request fails, verify that the Agent and your endpoint are both online before trying again.
```

Comment:
Still not 100% on case, but let's lowercase it for now. Its proper noun `Elastic Agent` {agent} is title case. 
```suggestion
TIP: If the request fails, verify that the agent and your endpoint are both online before trying again.
```